packages/components/icon/src/icon.tsx

```diff
@@ -34,7 +34,8 @@ export interface IconProps
  * @see Docs https://yamada-ui.com/components/media-and-icons/icon
  */
 export const Icon = forwardRef<IconProps, "svg">((props, ref) => {
-  const [styles, { size, ...mergedProps }] = useComponentStyle("Icon", props)
+  const [styles, { size, "aria-hidden": ariaHidden = true, ...mergedProps }] =
```

Comment:
The `"aria-hidden": ariaHidden = true` is not needed. Because it is included in `rest`.

Reply:
fixed
2f10ba6